### ISSUE TITLE
about 'to_csv' error

### ISSUE THREAD

**Issue by sorrowyn** [ReML-AI/MGTN#2]
In engine.py flie, 

`with open('out.csv', 'a') as f:
      f.write(self.state['out'][0] + ",")
      f.write(to_csv(self.state['target'].detach().cpu().numpy()[0]) + ',')
with open('out.csv', 'a') as f:
      f.write(to_csv(self.state['output'].detach().cpu().numpy()[0]) + '\n')`

The to_csv function is undefined

**Comment by sonvx** [comment]
Pls pull again. It should fix the problem.
